### DISCUSSION TITLE
Ошибка при выкладывании поста

### DISCUSSION THREAD

**Discussion by Katsumi666** [MrZillaGold/VK2Discord#541] [Q&A] [answered]
При попытке запуска бота и выкладывания поста в группе в ВК, выползает ошибка "webhook_url_invalit". Делала все по инструкции. С чем это может быть связано?
![d_7_eRKzbBw](https://user-images.githubusercontent.com/107765689/174450141-4d303f47-2828-4cb1-b9ad-362d41c10e6c.jpg)

**Comment by egorprnn** [comment, marked as answer]
Discord прекратил поддержку discordapp.com домена, используйте discord.com в вебхуках.
